chunk/schema.go

```diff
@@ -52,6 +52,9 @@ type IndexEntry struct {
 	// - If neither is set, must read all keys for that row.
 	RangeValuePrefix []byte
 	RangeValueStart  []byte
+
+	// Filters for querying
+	ValueEq []byte
```

Comment:
I'm not usually picky about names, I promise :-)

...but I would either call this `Value`, `ValueEqual`.